Shared/Model/Activity.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Activity.swift
+//  Fun Gen
+//
+//  Created by Julian Burrington on 4/23/22.
+//
+
+import Foundation
+
+struct Activity: Codable {
+    var id: String
+    var title: String
+    var category: Category
+    var author: User
+    var members: [User] = []
+    var options: [Option] = []
```

Comment:
How will the front end team know if there's an selected option, and if so, which?

Reply:
This might also be a case where we store an array of Firebase generated Strings that reference the Option objects. I think we want to be able to pull the title, author, member's, voteCount, or whatever from each object in ActivityViewModel for front end to use.

Since you have experience with this, does that sound like a good approach?

Reply:
I gave Option an id property

```swift
struct Option: Codable {
    /// Unique database ID
    var id: String
    ...
}
```

And now Activity holds an array of Option.id

```swift
struct Activity: Codable {
    ...
    /// Array of *Option.id*
    var options: [String] = []
    ...
}
```

Reply:
Do you know how to make a tuple conform to Codable? Seems like we would have to manually encode/decode to get this to work.

```swift
/// FIXME: Make this a tuple of (Option.ID, [User.ID])
/// Contains all ``Option``s suggested by ``members``
var options: [Option.ID] = []
```

Reply:
Let's just make it another struct then, maybe PollOption

Reply:
Did you still want to merge now and make this change at a later time?

Reply:
Just pushed, please take a look and let me know what you think

Reply:
Looks good!